babylon/build.boot

```diff
@@ -17,11 +17,12 @@
 
 (deftask package []
   (comp
-    (download :url "http://cdn.babylonjs.com/2-2/babylon.js")
```

Comment:
Could you try

```
(comp
  (download :url (format "https://raw.githubusercontent.com/BabylonJS/Babylon.js/v%s/dist/babylon.js" +lib-version+))
  (sift :move {#"babylon.js" "cljsjs/babylon/production/babylon.min.inc.js"})
  ...)
```

So we get versioned url. Also the file doesn't need to downloaded second time, no `.min.inc.js` file is needed as the same file can be used for both development and production builds.

Reply:
1. Sure, will do!
2. Yes that is redundant.

Reply:
Rebased and force pushed to a single commit.